### ISSUE TITLE
XLS: unknown encryption mechanism

### ISSUE THREAD

**Issue by AlexObukhoff** [qax-os/excelize#1673]
**Error open XLS file: unknown encryption mechanism**

**Steps to reproduce the issue:**
1. Debug method `excelize.OpenFile("file_example_XLS_5000.xls")`
2. Break on `github.com/xuri/excelize/v2@v2.8.0/excelize.go:186`
3. Check value of `err` variable

**Describe the results you received:**
Error open XLS file 

**Describe the results you expected:**
Sucessful open XLS file 

**Output of `go version`:**

```text
go version go1.21.1 linux/amd64
```

**Excelize version or commit ID:**

```text
github.com/xuri/excelize/v2@v2.8.0
```

**Environment details (OS, Microsoft Excel™ version, physical, etc.):**
System:
  Kernel: 5.15.0-84-generic x86_64 bits: 64 compiler: gcc v: 11.4.0 Desktop: Cinnamon 5.6.8
    tk: GTK 3.24.33 wm: muffin dm: LightDM Distro: Linux Min

Excel file: see attached.

**Comment by AlexObukhoff** [comment]
[file_example_XLS_5000.xls](https://github.com/qax-os/excelize/files/12724708/file_example_XLS_5000.xls)

**Comment by xuri** [comment]
Thanks for your issue. This duplicated with issues #44, #264, #368, #572 #1381. This library doesn't support the legacy XLS format currently. I closed this and merged it into issue #44. Please let me know if you have any questions to reopen this anytime.
